std_msgs/Header       header
uint64               time
geometry_msgs/Quaternion orientation
geometry_msgs/Vector3 angular_velocity
geometry_msgs/Vector3 LLA
geometry_msgs/Vector3 nedvel
geometry_msgs/Vector3 linear_acceleration
float32               dtime
float32[3]            dtheta
float32[3]            dvel
uint8                 fix
geometry_msgs/Vector3 gpsLLA
geometry_msgs/Vector3 gpsnedvel
